examples/kubernetes-istio/bookinfo-details-v1.yaml

```diff
@@ -49,7 +49,7 @@ specs:
         rules:
           http:
           - method: GET
-            path: "^/details/[0-9]*$"
+            path: "/details/[0-9]*"
```

Comment:
Out of curiosity, are the patterns auto-anchored or might this match anything that ends with `/details/[0-9]*`?

Reply:
With Envoy they’re supposed to be.